definitions/npm/lodash_v4.x.x/flow_v0.63.x-/test_lodash-v4.x.x.js

```diff
@@ -309,23 +309,28 @@ zipWith(["a", "b", "c"], [1, 2, 3]).map(([x, y]) => x * y);
  * _.isString
  */
 
-var boolTrue: true;
-var boolFalse: false;
-
-boolTrue = isString("foo");
-boolFalse = isString([""]);
-boolFalse = isString({});
-boolFalse = isString(5);
-boolFalse = isString(function(f) {
-  return f;
-});
-boolFalse = isString();
-boolFalse = isString(true);
-
-// $ExpectError
-boolFalse = isString("");
-// $ExpectError
-boolTrue = isString(undefined);
+() => {
+  const x = '';
+  !isString(x) && (x: empty);
+}
+() => {
+  const x = {};
+  isString(x) && (x: empty);
+}
+() => {
+  const x = 5;
+  isString(x) && (x: empty);
+}
+() => {
+  const x = () => {};
+  isString(x) && (x: empty);
+}
+() => {
+  const x: mixed = 'foo';
+  isString(x) && (x: string);
+  // $ExpectError
+  !isString(x) && (x: string);
+}
```

Comment:
Just use simple blocks, rather than functions.

```
{
  ...
}
```

But I like where you're going with it; the global scope nonsense in these tests make me sad.

Also: bonus points for writing out what is getting tested, TDD style.

Reply:
fixed